packages/blade-old/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @razorpay/blade-old
 
+## 1.0.0
+
+### Major Changes
+
+- c212fa0: Allow textinput right icon to be touchable
```

Comment:
Major release? 🙄 I thought we're just incrementing patch.
@kamleshchandnani Are we following semver here for breaking changes?

Reply:
@dhruvdutt we should follow semver here as well. This is a breaking change and since PG is using it too, we don't want to break things for them